doc/dev/developer_guide/dash-devel.rst

```diff
@@ -453,7 +453,7 @@ Start E2E tests by running::
 You can also start a cluster in development mode and later run E2E tests by running::
 
   $ ./src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh --dev-mode
-  $ # Work on your feature, bug fix, ...
+  $ # Add ``--expanded`` if you need to have a fully functional cluster.
```

Comment:
What does 'fully functional cluster' mean? 🤔 And clarifying that would be a good chance to explain what 'development mode' option provides too!

Reply:
Done.